jobmon_gui/src/components/navigation/PageNavigation.tsx

```diff
@@ -104,6 +108,8 @@ export default function PageNavigation({children}: PropsWithChildren) {
     const [open, setOpen] = useState(false);
     const navigate = useNavigate();
     const [drawerOpen, setDrawerOpen] = useState(false);
+    const {user} = useContext(AuthContext)
```

Comment:
Just asking out of curiosity - is there a reason to `useContext` over zustand for auth here?

Reply:
This is just how it was initially implemented in other apps, and I haven't found a compelling reason to change it.